src/vdaf/dummy.rs

```diff
@@ -0,0 +1,327 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Implementation of a dummy VDAF which conforms to the specification in [draft-irtf-cfrg-vdaf-06]
+//! but does nothing. Useful for testing.
+//!
+//! [draft-irtf-cfrg-vdaf-06]: https://datatracker.ietf.org/doc/draft-irtf-cfrg-vdaf/06/
+
+use crate::{
+    codec::{CodecError, Decode, Encode},
+    vdaf::{self, Aggregatable, PrepareTransition, VdafError},
+};
+use rand::random;
+use std::{fmt::Debug, io::Cursor, sync::Arc};
+
+type ArcPrepInitFn =
+    Arc<dyn Fn(&AggregationParam) -> Result<(), VdafError> + 'static + Send + Sync>;
+type ArcPrepStepFn = Arc<
+    dyn Fn(&PrepareState) -> Result<PrepareTransition<Vdaf, 0, 16>, VdafError>
+        + 'static
+        + Send
+        + Sync,
+>;
+
+/// Dummy VDAF that does nothing.
+#[derive(Clone)]
+pub struct Vdaf {
+    rounds: u32,
+    prep_init_fn: ArcPrepInitFn,
+    prep_step_fn: ArcPrepStepFn,
+}
+
+impl Debug for Vdaf {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Vdaf")
+            .field("prep_init_result", &"[redacted]")
```

Comment:
nit: I think these field names need to be updated (or dropped). IMO, also include `rounds`.

Reply:
Moot because `rounds` was moved into the prep step closure.